src/AutoMapper/Configuration/PathConfigurationExpression.cs

```diff
@@ -28,6 +28,11 @@ public void MapFrom<TSourceMember>(Expression<Func<TSource, TSourceMember>> sour
             MapFromUntyped(sourceExpression);
         }
 
+        public void Ignore()
+        {
+            PathMapActions.Add(pm => pm.Ignored = true);
```

Comment:
The code is simple enough, but there is some duplication here. We could just copy Ignored to PropertyMap.Ignored and handle both cases (PathMaps and PropertyMaps) in TryPopertyMap.